manifests/s/Synology/ChatClient/1.1.1-57/Synology.ChatClient.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Synology.ChatClient
+PackageVersion: 1.1.1-57
+PackageLocale: en-US
+Publisher: Synology Inc.
```

Comment:
```suggestion
Publisher: Synology
```

Reply:
Shouldn't the full company name be entered here?
If it's because it's already like this for the Synology Drive Client's 2.0.4 manifest, I changed it in the newer version (#19672)